data/mods/Magiclysm/monstergroups.json

```
@@ -189,7 +189,13 @@
     "monsters": [
       { "monster": "mon_orc_warrior", "freq": 100, "cost_multiplier": 5 },
       { "monster": "mon_orc_archer", "freq": 25, "cost_multiplier": 7 },
-      { "monster": "mon_orc_blood_warrior", "freq": 15, "cost_multiplier": 5 }
+      { "monster": "mon_orc_blood_warrior", "freq": 15, "cost_multiplier": 5 },
+      {
+        "type": "monstergroup",
+        "name": "GROUP_WORKSHOP",
+        "default": "mon_workshop_elf",
+        "monster": [ { "monster": "mon_workshop_elf", "freq": 100, "cost_multiplier": 2 } ]
```

I think you want this
```suggestion
        "type": "monstergroup",
        "name": "GROUP_WORKSHOP",
        "default": "mon_workshop_elf",
        "monsters": [ { "monster": "mon_workshop_elf", "freq": 100, "cost_multiplier": 2 } ]
```
Or this
```suggestion
        "monster": "mon_workshop_elf", "freq": 100, "cost_multiplier": 2
```

---

will try both to see if they work.

---

Both should work (though the first one is probably at least slightly wrong) - it just depends on if you want a new monstergroup here, or just an individual monster added to this monstergroup.

---

I think that they should be their own monstergroup as the previous group is orcs.

---

Oh, I totally misunderstood what you're trying to do then. None of this is valid, you somehow got this new monstergroup def stuck into te orc one.

---

Does that mean that orcs and workshop monstergroups are not separated properly? Is that the problem.